### ISSUE TITLE
package no longer installable?

### ISSUE THREAD

**Issue by epruesse** [bioconda/bioconda-recipes#10706]
```
conda install arb-bio

PackagesNotFoundError: The following packages are not available from current channels:

  - arb-bio
  - phyml==3.2.0
  - openmpi=2
```

Anyone have an idea what's going on there?

**Comment by epruesse** [comment]
Think I found the issue here: #3947

openmpi version 2 was deleted from bioconda but is not available from conda-forge

**Comment by metageni** [comment]
@epruesse 

Did you set-up the channels before trying to install arb-bio?  (see https://bioconda.github.io/ `Set up channels`)

I was able to install it the tool with no problem.

`conda create -n arb-bio -c bioconda arb-bio`
`source activate arb-bio`

`(arb-bio) $ arb_
arb_2_ascii               arb_dnapars               arb_help2xml              arb_name_server           arb_probe                 arb_remote                arb_trace
arb_2_bin                 arb_dnarates              arb_help_useredit.sh      arb_nexus2newick.awk      arb_proml                 arb_repair                arb_treegen
arb_a2ps                  arb_echo                  arb_ign                   arb_notify                arb_proto_2_xsub          arb_replace               arb_wait
arb_bootstrap             arb_edit4                 arb_installubuntu4arb.sh  arb_ntree                 arb_protpars              arb_rexec                 arb_wetc
arb_clean                 arb_export_rates          arb_launcher              arb_panic                 arb_pt_server             arb_rnacma                arb_who
arb_consensus_tree        arb_export_tree           arb_ludwig                arb_pars                  arb_raxml                 arb_sed                   
arb_convert_aln           arb_fastdnaml             arb_macsetup              arb_phylo                 arb_read_tree             arb_sleep                 
arb_db_server             arb_flush_mem             arb_message               arb_phyml                 arb_readlink              arb_textedit              
arb_dist                  arb_gene_probe            arb_naligner              arb_primer                arb_readseq               arb_textprint `

**Comment by epruesse** [comment]
@metageni Thanks for looking into this! :)

You can't reproduce the error anymore because I fixed it about an hour ago (see 074ec67f70da615e08c0457c5a4e0e4959d95bd5 merging #10707). I hadn't closed this yet because the packages were still being built. 

The culprit was in the end that `arb-bio` requires `phyml=3.2.0` which requires `openmpi 2.*` which was moved to `bioconda-legacy` in favor of conda-forge's `openmpi`, which is (now) only available in version 3 and above, the older versions having been assigned a label other than `main`.

To make matters worse, the author of PhyML deleted the 3.2.0 and 3.3.2017* versions from the GitHub releases page, making it impossible to re-compile. I'm now using the sources archived by Debian in the package. 

Long story short - rebuilding `phyml` against `openmpi 3` fixed it. 

@bgruening It might be worth consideration to *always* copy sources to cargo-port and then use those for building. At the very least compiled GPL software where we are legally required to keep a copy of the source for distribution as long as we distribute the binaries should go this route.
